### ISSUE TITLE
Error parsing standard object xml

### ISSUE THREAD

**Issue by arbokrad** [cesarParra/apexdocs#203]
We are seeing failures parsing standard object XML on v3.4.0 of apexdocs when trying to run `markdown`.

For the following object xml:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
    <actionOverrides>
        <actionName>CallHighlightAction</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>CancelEdit</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>Delete</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>Edit</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>EmailHighlightAction</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>EnableCustomerPortalUser</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>List</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>ListClean</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>New</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>RequestUpdate</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>SaveEdit</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>SmsHighlightAction</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>Tab</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>View</actionName>
        <content>Account_Record_Page</content>
        <formFactor>Large</formFactor>
        <skipRecordTypeSelect>false</skipRecordTypeSelect>
        <type>Flexipage</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>ViewCustomerPortalUser</actionName>
        <type>Default</type>
    </actionOverrides>
    <actionOverrides>
        <actionName>WebsiteHighlightAction</actionName>
        <type>Default</type>
    </actionOverrides>
    <compactLayoutAssignment>Account_Compact_Layout</compactLayoutAssignment>
    <enableEnhancedLookup>true</enableEnhancedLookup>
    <enableFeeds>true</enableFeeds>
    <enableHistory>false</enableHistory>
    <searchLayouts>
        <customTabListAdditionalFields>ACCOUNT.NAME</customTabListAdditionalFields>
        <customTabListAdditionalFields>ACCOUNT.ADDRESS1_CITY</customTabListAdditionalFields>
        <customTabListAdditionalFields>ACCOUNT.PHONE1</customTabListAdditionalFields>
        <lookupDialogsAdditionalFields>ACCOUNT.NAME</lookupDialogsAdditionalFields>
        <lookupDialogsAdditionalFields>CORE.USERS.ALIAS</lookupDialogsAdditionalFields>
        <lookupDialogsAdditionalFields>ACCOUNT.TYPE</lookupDialogsAdditionalFields>
        <lookupPhoneDialogsAdditionalFields>ACCOUNT.NAME</lookupPhoneDialogsAdditionalFields>
        <lookupPhoneDialogsAdditionalFields>CORE.USERS.ALIAS</lookupPhoneDialogsAdditionalFields>
        <lookupPhoneDialogsAdditionalFields>ACCOUNT.TYPE</lookupPhoneDialogsAdditionalFields>
        <lookupPhoneDialogsAdditionalFields>ACCOUNT.PHONE1</lookupPhoneDialogsAdditionalFields>
        <searchResultsAdditionalFields>ACCOUNT.NAME</searchResultsAdditionalFields>
        <searchResultsAdditionalFields>ACCOUNT.PHONE1</searchResultsAdditionalFields>
        <searchResultsAdditionalFields>CORE.USERS.ALIAS</searchResultsAdditionalFields>
    </searchLayouts>
    <sharingModel>ReadWrite</sharingModel>
</CustomObject>
```

We get the following error:
```pwsh
10/25/2024, 2:25:52 PM: Generating markdown documentation...
10/25/2024, 2:27:10 PM: ❌ An error occurred while processing the request: Error(s) occurred while parsing files. Please review the following issues:,Source file: force-app\main\default\objects\Account\Account.object-meta.xml
Invalid SObject metadata
```

This, unfortunately, tells me almost nothing about whats really happening to fail this. The metadata deploys to the corresponding Salesforce org without issue. Any thoughts?

Edit: Alternatively, is there a way to tell it to just not do object xml at all?

**Comment by cesarParra** [comment]
Thanks for reporting this @arbokrad .

It seems like my validation was a bit aggressive and assumed that there were some required properties in the object xml (in this case, the label is missing, but it is still valid).

I'll remove any of those assumptions from the validation and also improve the error messages that are displayed.

**Comment by cesarParra** [comment]
I just released a new version (3.4.2) which hopefully resolves the issue, but please let me know if you run into anything else.

In that version, you should also be able to exclude XML by using the `excludes` configuration property (either in the `package.json` or a standalone `apexdocs.config.js`, e.g using package.json:

```js
// package.json file
...
  "apexdocs": {
    "exclude": ["**/*.object-meta.xml", "**/*.field-meta.xml"]
  }
...
```
